uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -529,8 +529,8 @@ sub identify_scan_db {
 
     
     # get the list of protocols for a site their scanner and subproject
-    $query = "SELECT Scan_type, ScannerID, Center_name, TR_range, TE_range, TI_range, slice_thickness_range, xspace_range, yspace_range, zspace_range,
-              xstep_range, ystep_range, zstep_range, time_range, series_description_regex
+    $query = "SELECT Scan_type, ScannerID, Center_name, TR_min, TR_max, TE_min, TE_max, TI_min, TI_max, slice_thickness_min, slice_thickness_max, xspace_min, xspace_max, yspace_min, yspace_max, zspace_min, zspace_max,
+              xstep_min, xstep_max, ystep_min, ystep_max, zstep_min, zstep_max, time_min, time_max, series_description_regex
```

Comment:
Would be nice to have an error message if the query fails to tell the user that they need to update their `mri_protocol` table. What do you think?
![screen shot 2018-05-28 at 4 22 04 pm](https://user-images.githubusercontent.com/1402456/40628497-63d32fb4-6293-11e8-8411-ac368d08cab0.png)

Reply:
sounds good!